sdk/tests/invariant.test.ts

```diff
@@ -146,46 +147,69 @@ describe('invariant', async () => {
     const isUpperTickInitialized = await invariant.isTickInitialized(account, poolKey, 10n)
     assert.deepEqual(isUpperTickInitialized, true)
   })
+  it('create pool', async () => {
+    if (!token0.contract?.address || !token1.contract?.address || !invariant.contract?.address) {
+      throw new Error()
+    }
+
+    const feeTier = newFeeTier({ v: 10000000000n }, 1)
+    await invariant.addFeeTier(account, feeTier)
+    const addedFeeTierExists = await invariant.feeTierExist(account, feeTier)
+    assert.deepEqual(addedFeeTierExists, true)
 
-  // //TODO: needs PR with FeeTiers
-  // it('create pool', async () => {
-  //   const { api, account } = await init()
-
-  //   const invariantData = await getDeploymentData('invariant')
-  //   const invariant = new Invariant(api, Network.Local)
-
-  //   const initFee = { v: 10 }
-  //   const invariantDeploy = await invariant.deploy(
-  //     account,
-  //     invariantData.abi,
-  //     invariantData.wasm,
-  //     initFee
-  //   )
-  //   await invariant.load(invariantDeploy.address, invariantData.abi)
-
-  //   const token0: string = '5H79vf7qQKdpefChp4sGh8j4BNq8JoL5x8nez8RsEebPJu9D'
-  //   const token1: string = '5DxazQgoKEPMLqyUBRpqgAV7JnGv3w6i4EACTU8RDJxPHisH'
-  //   const fee: Percentage = { v: 100n }
-  //   const feeTier: FeeTier = newFeeTier(fee, 1)
-  //   const initSqrtPrice: SqrtPrice = { v: 1000000000000000000n }
-  //   const initTick = 1n
-
-  //   const createPoolResult = await invariant.createPool(
-  //     account,
-  //     token0,
-  //     token1,
-  //     feeTier,
-  //     initSqrtPrice,
-  //     initTick
-  //   )
-  //   await sleep(1000)
-
-  //   console.log(createPoolResult)
-
-  //   const result = await invariant.getPool(account, token0, token1, feeTier)
-  //   console.log(result)
-
-  //   const pools = await invariant.getPools(account)
-  //   console.log(pools)
-  // })
+    const initSqrtPrice: SqrtPrice = { v: 1000000000000000000000000n }
+    const initTick = 0n
+
+    await invariant.createPool(
+      account,
+      token0.contract.address.toString(),
+      token1.contract.address.toString(),
+      feeTier,
+      initSqrtPrice,
+      initTick
+    )
+    const pools = await invariant.getPools(account)
```

Comment:
check also specific pool state. There is message to access pool state via `position_index`